mautic/mautic
# TPROD-318 - Converting PHP templates for the IntegrationsBundle to Twig

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-318



#### Description:

Converts PHP templates for the IntegrationsBundle to Twig.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. As there is no publicly available plugin to test this with (yet) then I guess we'll have to trust myself and our CI checks that it will work 🤷  There is quite great chance that this code have a bug or 2 in it as other PRs that refactor templates. However, we'll have to take the risk to move forward and the companies that use this bundle will have to fix the potential bugs as they start converting their integrations to M5. 
3. Please code-review.
